cli/azd/internal/scaffold/scaffold.go

```diff
@@ -210,6 +210,15 @@ func preExecExpand(spec *InfraSpec) {
 				Secret: true,
 			})
 	}
+	if spec.DbMySql != nil {
+		spec.Parameters = append(spec.Parameters,
+			Parameter{
+				Name:   "mysqlDatabasePassword",
+				Value:  "$(secretOrRandomPassword ${AZURE_KEY_VAULT_NAME} mysqlDatabasePassword)",
```

Comment:
```suggestion
				Value:  "$(secretOrRandomPassword ${AZURE_KEY_VAULT_NAME} MYSQL-PASSWORD)",
```

Feel free to update the one for postgres as wel, since I'm planning to fix these later

Reply:
lowercased everything